src/test/java/AlgorithmHandler/algorithms/BasicAlgorithm.java

```diff
@@ -0,0 +1,7 @@
+package AlgorithmHandler.algorithms;
```

Comment:
Open to style recommendations, these are the algorithmia algorithm examples that we're testing in the `tests` directory

Reply:
Should still follow java package name conventions. Also I usually like to put tests in the same package as the thing they are testing (it can sometimes be helpful to be able to access protected members and functions).

Reply:
I can do that, would it make sense to have a package for each algo/test combo instead? I want to avoid clutter here as it can be a bit noisy.

Reply:
Let me know what you think about the current change (algorithmia.algorithm.Handler)